html/contact.html

```diff
@@ -349,7 +349,7 @@
         }
 
         h1{
-            color: #000;
+            color: orange;
```

Comment:
Evgheni! Please Don't change already existing styles. If you need to make any changes, please do so, at the end of the file WITH a specific selector like a unique class.